tests/client/amo/components/TestLanguagePicker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+
+import LanguagePicker from 'amo/components/LanguagePicker';
+import { getFakeI18nInst } from 'tests/client/helpers';
+import I18nProvider from 'core/i18n/Provider';
+
+
+describe('LanguagePicker', () => {
+  function renderLanguagePicker({ ...props }) {
+    return findRenderedComponentWithType(renderIntoDocument(
+      <I18nProvider i18n={getFakeI18nInst()}>
```

Comment:
You could use `FooterBase` without the `translate()` call though. I actually think we should update translate so props override the context though.


```jsx
  function renderFooter({ ...props }) {
    return findRenderedComponentWithType(renderIntoDocument(
      <FooterBase {...props} />
    ), FooterBase);
  }
```